README.md

```diff
@@ -20,59 +24,107 @@ If your Internet access is expensive/rare/slow/unreliable/watched/censored, you
 
 ## Usage
 
-It uses ZIM files that you can download from https://download.kiwix.org/zim/
+Install "Kiwix JS" from your browser's add-on store, or you can get it from http://download.kiwix.org/release/browsers/. Alternatively, bookmark or
```

Comment:
We should say here that the add-on store is the much preferred way to install it (because of automatic updates). The other ways to install are for very specific use-cases

Reply:
OK, will add that.